tidy3d/components/data/sim_data.py

```diff
@@ -819,7 +831,7 @@ def plot_scalar_array(
         y_coord_values = field_data.coords[y_coord_label]
         ax.set_xlim(min(x_coord_values), max(x_coord_values))
         ax.set_ylim(min(y_coord_values), max(y_coord_values))
-
+        self.__dict__["field_data_"] = field_data
```

Comment:
what does this do? was it left in by mistake?

Reply:
At some point, I wanted to access the field data, so I did this ugly workaround. I forgot to remove it.